### PR TITLE
Re-add gauge from earlier versions to stake page

### PR DESCRIPTION
**Pull request by exitonoff**
Addresses issue https://github.com/SharedStake/SharedStake-ui/issues/52

Found the earlier gauge:
image

Changed some colours to make it fit to the site. Changed the calculation of ETH deposited to contract to reflect actual amount of ETH deposited (earlier it showed the shares minted / max shares). Added it to Stake-page:
image

Some issues:
- it doesn't seem to work on Firefox...though it seems that there's some more problems with FF discussed at #bug-reports in Discord
  - tested with Chrome and Brave and was working fine
- didn't seem to work on mobile
  - I haven't used wallets on mobile (only Metamask on desktop), so I'm not sure how to fix this
- the gauge had previosly explanation text:
> This shows the total Eth amount that is staked to SharedStake so far.
> 320 is required (10 validators) for the first deposit to Eth2 event which locks it up. We will increase this limit to 3200, then 32000 (1000 validators) for subsequent deposit events.
>When Eth is deposited into the SharedDeposit contract, a Validator-Share-Eth2 token (vEth2) is minted. Redeemable for the deposited Eth.
- I edited it a bit to:
> Total of {{ ethDeposited }}Eth has been staked to SharedStake so far.
> 32100Eth is required (for 1000 validators) for the third deposit to ETH2 event which locks it up.
> When Eth is deposited into the SharedDeposit contract, a Validator-Share-Eth2 token (vEth2) is minted. Redeemable for the deposited Eth.
- what should it be?

### CODE REVIEW

File: src/components/Stake/StakeGauge.vue [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="gauge">
+    <div class="explanation">
+      <div class="title">SharedStaked vEth2</div>
+      <div class="content">
+        Total of {{ ethDeposited }}Eth has been staked to SharedStake so far.
+        <br />
+        32100Eth is required (for 1000 validators) for the third deposit to ETH2 event
```
Comment (sharedStake-dev) [inline src/components/Stake/StakeGauge.vue:8]: There's actually nothing more than 32 eth needed for the 3rd deposit. The tardiness has been due to figuring out more cloud providers and regions and issues with AWS. 

- Can we therefore remove the comment about the 3rd deposit? 
- Can we change `Eth` to `ETH`? 
- Can we have the base value of 32100 be dynamic? (from teh contract it's costPerValidator * numValidators)

**Review by sharedStake-dev — CHANGES_REQUESTED** [review]

**Comment by exitonoff** [comment]
I added commit to update Earn-page's buttons' label texts, as suggested in https://github.com/SharedStake/SharedStake-ui/issues/61. Thought it would be fine to add it to this, if this PR is going to be merged anyways soon.

File: src/components/Stake/Stake.vue [hunk 2]
```diff
@@ -190,43 +106,66 @@ export default {
...
+
+          return;
         }
-        this.Damount = this.BNamount.dividedBy(1e18);
-      } else {
+      }
+
+      // Is depositing vETH2
+      if (!this.isDeposit) {
+        console.log("SASDSA")
```
Comment (sharedStake-dev) [inline src/components/Stake/Stake.vue:142]: lets get rid of the debug logs

**Review by sharedStake-dev — CHANGES_REQUESTED** [review]

File: src/components/Stake/Stake.vue (hunk 2 above)
Comment (exitonoff) [reply src/components/Stake/Stake.vue]: Whoopsie, removed them. 🚯

**Pull request merged**